src/main/java/sirius/biz/jobs/params/AmountParameter.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.jobs.params;
+
+import sirius.kernel.commons.Amount;
+import sirius.kernel.commons.Value;
+import sirius.kernel.nls.NLS;
+
+import java.util.Optional;
+
+/**
+ * Provides a parameter which accepts {@link Amount amounts}.
+ */
+public class AmountParameter extends Parameter<Amount, AmountParameter> {
```

Comment:
should't we add a withSuffix(...) which adds a suffix (auto-i18nd) to the field?